### PR TITLE
[PR-5835] Staging for Docs

### PR DESCRIPTION
**Pull request by pustovalov**
Also need to add new git remote:

`git remote add staging https://github.com/talkable/void-talkable-docs.git`

## Related Stories

- [PR-5835](https://talkable.atlassian.net/browse/PR-5835)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -72,3 +72,21 @@ deploy:
 	git commit -m "Generated gh-pages for `git log master -1 --pretty=short --abbrev-commit`" && git push origin gh-pages ; git checkout master
 	@echo
 	@echo "Deployment finished. Check updated docs at http://docs.talkable.com"
+
+stage:
+	git checkout gh-pages-void
+	git pull
+	rm -rf $(BUILDDIR) $(SOURCEDIR)
+	git checkout void $(GH_PAGES_SOURCES)
```
Comment (zhuravel) [inline Makefile]: I would make another task to setup remotes, e.g.:
```
setup:
    git remote add staging git@github.com:talkable/void-talkable-docs.git
```

File: Makefile [hunk 2]
```diff
@@ -72,3 +75,21 @@ deploy:
...
+	git reset HEAD
+	make html
+	(cd $(BUILDDIR)/html && tar c ./) | (cd ./ && tar xf -)
+	rm -rf $(GH_PAGES_SOURCES) $(BUILDDIR) .buildinfo
+	echo '' > .nojekyll
+	echo "/build\n/.bundle" > .gitignore
+	echo 'void-docs.talkable.com' > CNAME
+	echo "User-agent: *\nDisallow: /" > robots.txt
+	git add -A
+	git commit -m "Generated gh-pages-void for `git log void -1 --pretty=short --abbrev-commit`" && git push -f staging gh-pages-void:gh-pages ; git checkout void
```
Comment (zhuravel) [inline Makefile]: "Generated gh-pages-void" → "Generated gh-pages"

File: Makefile (hunk 1 above)
Comment (zhuravel) [inline Makefile]: What is "void" branch?

File: Makefile [hunk 3]
```diff
@@ -72,3 +75,21 @@ deploy:
 	git commit -m "Generated gh-pages for `git log master -1 --pretty=short --abbrev-commit`" && git push origin gh-pages ; git checkout master
 	@echo
 	@echo "Deployment finished. Check updated docs at http://docs.talkable.com"
+
+stage:
```
Comment (vitalikdanchenko) [inline Makefile]: I would extract common code from `stage` and `deploy` parts

**Pull request closed without merging**